test/commands/exercise/representation/sync_to_search_index_test.rb

```diff
@@ -2,57 +2,57 @@
 
 class Exercise::Representation::SyncToSearchIndexTest < ActiveSupport::TestCase
   test "indexes representation" do
-    skip
-    user = create :user, id: 7, handle: 'jane'
     track = create :track, id: 11, slug: 'fsharp', title: 'F#'
     exercise = create(:practice_exercise, id: 13, slug: 'bob', title: 'Bob', track:)
-    solution = create :practice_solution,
-      id: 17,
-      num_stars: 3,
-      num_loc: 55,
-      num_views: 20,
-      num_comments: 2,
-      user:,
-      exercise:,
-      published_iteration_head_tests_status: :failed
-    submission = create :submission, solution:, tests_status: :passed
-    create :submission_file, submission:, content: "module LogLineParser"
-    iteration = create(:iteration, submission:)
+    content = "module LogLineParser"
+    num_published_solutions = 20
+    reputation = 1234
+    num_loc = 42
+
+    solutions = Array.new(2).map do
+      create(:practice_solution, :published, published_iteration_head_tests_status: :passed).tap do |solution|
+        submission = create(:submission, solution:)
+        create(:iteration, submission:)
+      end
+    end
```

Comment:
Maybe create a non-published solutions before this code block to sanity check that only published solutions are considered?